dom5.ts

```diff
@@ -403,6 +403,17 @@ export function nodeWalkPrior(node: Node, predicate: Predicate): Node|
   return undefined;
 }
 
+export function nodeWalkParent(node: Node, predicate: Predicate): Node|undefined {
```

Comment:
Recommend instead `nodeWalkAncestors`

(The naming conventions in this file are bad in general I think. `walk` isn't right, I think it should be `find`, and `node` is extraneous, but probably better to be locally consistent for now)